### PR TITLE
Fix counting of trackers

### PR DESCRIPTION
**Pull request by jonathanKingston**
Ensure we count broken site trackers and displaying this to users.

https://app.asana.com/0/0/1200517965702747/f

**Reviewer:** @sammacbeth 

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/background/redirect.es6.js [hunk 1]
```diff
@@ -154,9 +144,9 @@ function handleRequest (requestData) {
             }
 
             browserWrapper.notifyPopup({ updateTabData: true })
-
-            // Block the request if the site is not whitelisted
-            if (!thisTab.site.whitelisted && tracker.action.match(/block|redirect/)) {
+            const allowListed = thisTab.site.whitelisted || thisTab.site.isBroken;
```
Comment (sammacbeth) [inline shared/js/background/redirect.es6.js:147]: Can we not use `thisTab.site.isAllowlisted` directly here?

File: shared/js/ui/templates/shared/tracker-networks-text.es6.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ module.exports = function (site, isMajorNetworksCount) {
 
 function trackersBlockedOrFound (site, trackersCount) {
     let msg = ''
-    if (site && (site.isWhitelisted || trackersCount === 0)) {
+    if (site && site.isAllowlisted || trackersCount === 0) {
```
Comment (sammacbeth) [inline shared/js/ui/templates/shared/tracker-networks-text.es6.js:20]: So this means that when `site` is falsy we also say "Found", while before we said "Blocked", right? In which cases is `site` falsy? I guess one example might be when the extension reloads while a tab is open. Will we not be blocking requests for the tab in that case?

File: shared/js/ui/views/site.es6.js [hunk 3]
```diff
@@ -17,6 +17,7 @@ function Site (ops) {
 
     // cache 'body' selector
     this.$body = window.$('body')
+    this.$protectionRow = window.$('body')
```
Comment (sammacbeth) [inline shared/js/ui/views/site.es6.js:20]: This assignment seems to be redundant, `this.$protectionRow` is identical to `this.$body`

File: shared/scss/popup.scss [hunk 4]
```diff
@@ -167,7 +167,7 @@ body {
             max-height: 19px;
             overflow: hidden;
 
-            .is-disabled & {
+            .is-disabled &, &.is-disabled {
```
Comment (sammacbeth) [inline shared/scss/popup.scss:170]: Why do we need both of these versions?

**Review by sammacbeth — CHANGES_REQUESTED** [review]
A few comments, and you have some lint errors to fix.

File: shared/js/ui/templates/shared/tracker-networks-text.es6.js (hunk 2 above)
Comment (jonathanKingston) [reply shared/js/ui/templates/shared/tracker-networks-text.es6.js:20]: We should be at least once the tds list is loaded. I think this has the potential to be the case after install, I'm not super familiar with when we trigger `site` calculation, I think it's on navigation from memory.

I've made the brackets match how they were anyway.

**Review by sammacbeth — APPROVED** [review]

**Pull request merged**